### DISCUSSION TITLE
Unable to Access Seq Web Interface on Windows Server

### DISCUSSION THREAD

**Discussion by BondByteBlaster** [datalust/seq-tickets#2058] [Help]
Hi, 

I am encountering an issue where I'm unable to access the Seq web interface on a Windows Server. This setup was functioning correctly acouple of month ago, but recently the web page became inaccessible. Below are the steps I've taken to troubleshoot and resolve the issue, but so far, I've had no success:

- Service Restart: I've restarted the Seq service multiple times.
- Running Seq in CMD: I ran the Seq service in the command line to monitor for any unusual logs, but nothing out of the ordinary was noted.
- Firewall Configuration: I disabled the firewall to rule out any network blockages.
- Protocol Verification: Attempted to access Seq via both HTTP and HTTPS.
- Port Accessibility: Verified that the designated port (5341) is open.
- Browser Tests: Tried accessing Seq from both Chrome and Edge on the server, as well as from a remote computer.
- Reinstallation: Recently uninstalled and reinstalled the latest version of Seq. The service starts without issues, and I can see in the logs that it's running, but the web interface remains inaccessible.
Additional details:

**Server OS:** Windows Server 2022 Datacenter Azure Edition
**Seq Version:** Seq-2023.4.10219.msi
**Error Messages:** 
This site can’t be reachedThe connection was reset.
Try:

Checking the connection
[Checking the proxy and the firewall](chrome-error://chromewebdata/#buttons)
[Running Windows Network Diagnostics](javascript:diagnoseErrors())
ERR_CONNECTION_RESET
Check your Internet connection
Check any cables and reboot any routers, modems, or other network devices you may be using.
Allow Chrome to access the network in your firewall or antivirus settings.
If it is already listed as a program allowed to access the network, try removing it from the list and adding it again.
If you use a proxy server…
Check your proxy settings or contact your network administrator to make sure the proxy server is working. If you don't believe you should be using a proxy server: Go to the Chrome menu > Settings > Show advanced settings… > Change proxy settings… > LAN Settings and deselect "Use a proxy server for your LAN".

**Telnet Test:** Successfully connected to port 5341 via telnet, indicating the port is open and listening.
I have not observed any certificate-related warnings or errors that would suggest an SSL/TLS issue. The lack of any evident errors in the logs or during service startup is particularly puzzling.

I would greatly appreciate any guidance or suggestions you might have to resolve this issue.

Best regards

**Comment by nblumhardt** [comment]
Hi @BondByteBlaster! Thanks for getting in touch.

I'm not aware of anything Seq-specific that would cause this kind of behavior, at least at first glance, unfortunately.

Do you have a supported Seq subscription? If so, can you please drop us a line via `support@datalust.co` and reference this ticket, and we'll do our best to help out.

Best regards,
Nick

**Comment by BondByteBlaster** [comment]
Thanks,

The service seems to be running fine but the webpage is not accessable at all. Maybe I should try to install the docker version instead, otherwize it seems like I have tested everything else.

I dont have a SEQ subscription yet :( 

Regards

**Comment by BondByteBlaster** [comment]
The problem was an invalid certificate, the server was set to not show any certificate validation problems, just to block so thats why I got "connection was reset". After fixing the certificate it worked fine again.

**Reply by nblumhardt** [reply]
That's great, thanks for letting us know.
